python-rhsm/src/rhsm/connection.py

```diff
@@ -1405,6 +1405,20 @@ def getSubscriptionList(self, owner_key):
         results = self.conn.request_get(method)
         return results
 
+    def updateSubscriptionList(self, owner_key, auto_create_owner=False, lazy_regen=True):
```

Comment:
Let's not specify the defaults for the parameters `auto_create_owner` and `lazy_regen`. Instead, in python-rhsm, let the params default to `None`... This way the defaults are specified in only one place (server-side, in candlepin code).